styles.css

```diff
@@ -1,9 +1,33 @@
-/* .main-page-form { */
+* {
+  font-family: 'Open Sans';
+}
+/* .container{
+  display: flex;
+} */
+
+.filter-starred-ideas {
+  border: 2px solid white;
+}
+.sidebar {
```

Comment:
Looks good so far, maybe we should try to use flex box in here somehow. I'll do some research